### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by TechnoAman**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`
aman1

### CODE REVIEW

**Comment by TechnoAman** [comment]
Here I am added the link to the blog 
title, img author name

File: _data/blogs.yml [hunk 1]
```diff
@@ -191,4 +191,9 @@
 - url: https://saarthakchaturvedi05.wordpress.com/2017/12/24/interview-with-long-term-open-source-contributors/
   title: Interview with Two Long-Term Open Source Contributors
   img: saarthakchaturvediinterview.jpg
-  author_name: Saarthak Chaturvedi
\ No newline at end of file
+  author_name: Saarthak Chaturvedi
+  
+- url: https://amantiwari123.wordpress.com/2017/12/27/my-journey-to-google-code-in-gci-2017-18/
+  title: My Journey to Google Code-In With FOSSASIA (2017-18)
+  img: Aman1.jpg
```
Comment (0x48piraj) [inline _data/blogs.yml:198]: Where's the `img` ?

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
The image is not present. :-1:

**Review by nikit19 — CHANGES_REQUESTED** [review]
You have to send one pull request which has the code that you have written here and also the image that you have submitted in the other pull request.

**Pull request closed without merging**